docs/submit-opentargets-batch.md

```diff
@@ -222,54 +245,49 @@ cd ${CODE_ROOT} && ${BSUB_CMDLINE} \
   -e ${BATCH_ROOT}/trait_mapping/trait_names_to_ontology_mappings.tsv \
   -g ${BATCH_ROOT}/gene_mapping/ot_mapping_result.out \
   -j ${BATCH_ROOT}/clinvar/clinvar.filtered.json.gz \
-  --ot-schema ${BATCH_ROOT}/evidence_strings/opentargets-$VERSION.json \
+  --ot-schema ${BATCH_ROOT}/evidence_strings/opentargets-${OT_SCHEMA_VERSION}.json \
   --out ${BATCH_ROOT}/evidence_strings/
 ```
 
-This outputs multiple files, including the evidence strings (`evidence_strings.json`) for submitting to OpenTargets and the file of trait mappings for submitting to ZOOMA (`eva_clinvar.txt`).
+This outputs multiple files, including the evidence strings (`evidence_strings.json`) for submitting to Open Targets and the file of trait mappings for submitting to ZOOMA (`eva_clinvar.txt`).
 
-Generated evidence strings must be additionally validated using tool provided by OpenTargets. _Note: as of August 2019, there is a problem running `opentargets_validator` module using Python 3; as a workaround you can install and run it locally using Python 2.__
+Generated evidence strings must be additionally validated using tool provided by Open Targets. _Note: as of August 2019, there is a problem running `opentargets_validator` module using Python 3; as a workaround you can install and run it locally using Python 2. To solve this, Python version needs to be updated to at least 3.6.__
```

Comment:
so it would work for us if we used python3.6 in this pipeline? Do you know if there's breaking changes?

Reply:
Yes, if we used Python 3.6 or newer, then it wouldn't be an issue. I don't know if there are breaking changes—this will need to be tested. I created an issue to address this: https://www.ebi.ac.uk/panda/jira/browse/EVA-1797